test/MyFirstApiConsume.test.js

```diff
@@ -0,0 +1,53 @@
+const axios = require("axios");
+const { expect } = require("chai");
+const { StatusCodes } = require("http-status-codes");
+
+describe("First Api Tests", () => {});
+
+it("Consume GET Service", async () => {
+    const response = await axios.get("https://httpbin.org/ip");
+
+    expect(response.status).to.equal(StatusCodes.OK);
+    expect(response.data).to.have.property("origin");
+});
+
+it("Consume GET Service with query parameters", async () => {
+    const query = {
+        name: "John",
+        age: "31",
+        city: "New York",
+    };
+
+    const response = await axios.get("https://httpbin.org/get", { query });
+
+    expect(response.status).to.equal(StatusCodes.OK);
+    expect(response.config.query).to.eql(query);
+});
+
+it("Consume HEAD Service", async () => {
+    const response = await axios.head("https://httpbin.org/anything");
+
+    expect(response.status).to.equal(StatusCodes.OK);
+    expect(response.headers).to.have.property("date");
+});
+
+it("Consume PATCH Service", async () => {
+    const response = await axios.patch("https://httpbin.org/anything");
+
+    expect(response.status).to.equal(StatusCodes.OK);
+    expect(response.data).to.have.property("data");
+});
+
+it("Consume PUT Service", async () => {
+    const response = await axios.put("https://httpbin.org/anything");
+
+    expect(response.status).to.equal(StatusCodes.OK);
+    expect(response.data).to.have.property("url");
+});
+
+it("Consume DELETE Service", async () => {
+    const response = await axios.delete("https://httpbin.org/anything");
+
+    expect(response.status).to.equal(StatusCodes.OK);
+    expect(response.data).to.have.property("method");
+});
```

Comment:
For the tests of PATCH, PUT and DELETE methods, please send a request body and verify that endpoints return it correctly.

Reply:
Ok, I'm going to make a body to that methods. With and add . + commit -m + push it's enough to update the file or i have to create another PR?

Reply:
updating this PR will be fine, you don't need to create new prs for this section